src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportFilters/ReportFilters.tsx

```diff
@@ -214,6 +236,25 @@ export const ReportFilters = ({
                 data-testid={`${dataTestid}-versions`}
               />
             </Box>
+            <Box sx={{ width: '24rem', ml: theme.spacing(2.4) }}>
```

Comment:
Nit: Just wondering if it's needed to prescribe a static width here. Some localized strings may not be suited to 240 pixels, and/or the parent flexbox may not always respect this value based on viewport constraints. If the idea is to prevent word wrapping, I'd do the following instead (also you can use the shorthand instead of calling `theme.spacing` here):

```suggestion
            <Box sx={{ whiteSpace: 'nowrap', ml: 2.4 }}>
```

Reply:
No reason at all, was basing it off the other components in this, but happy to accept this 👍